### PR TITLE
[11] [MIG] Account Analytic No Lines

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
Migration of account_analytic_no_lines from 10.0 to 11.0.
#133 

Migration to 12.0 : #191

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by ThomasBinsfeld** [comment]
@pedrobaeza Done

**Comment by pedrobaeza** [comment]
Thanks!

**Review by LoisRForgeFlow — APPROVED** [review]
Code LGTM :+1:

File: account_analytic_no_lines/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+{
+    'name': 'Account Analytic No Lines',
+    'summary': """
+        Hide analytics lines and disable
+        their generation from a move line.""",
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu/',
```
Comment (nikul-serpentcs) [inline account_analytic_no_lines/__manifest__.py]: @ThomasBinsfeld  replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

File: account_analytic_no_lines/views/account_move_line.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+-->
+
+<odoo>
+
+<record id="view_move_line_form" model="ir.ui.view">
+    <field name="name">hide_analytic_lines.account.move.line.form</field>
+    <field name="model">account.move.line</field>
+    <field name="inherit_id" ref="account.view_move_line_form"/>
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='analytic_line_ids']/.." position="attributes">
```
Comment (nikul-serpentcs) [inline account_analytic_no_lines/views/account_move_line.xml:14]: Directly use `field` here

**Review by nikul-serpentcs — COMMENTED** [review]
Small Change

File: account_analytic_no_lines/__manifest__.py (hunk 1 above)
Comment (ThomasBinsfeld) [reply account_analytic_no_lines/__manifest__.py]: Done

File: account_analytic_no_lines/views/account_move_line.xml (hunk 2 above)
Comment (ThomasBinsfeld) [reply account_analytic_no_lines/views/account_move_line.xml:14]: This xpath doesn't target the field but the group just before (which has no name).

**Comment by alexis-via** [comment]
The migration script to change the XMLID of the res.group from hide_analytic_entries to show_analytic_entries was missing

**Pull request closed without merging**